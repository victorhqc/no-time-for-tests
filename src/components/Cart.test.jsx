import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

describe('Cart', () => {
  it('renders', () => {});

  it('Should tell when cart is empty', () => {});

  it('render items from cart', () => {});

  it('render total amount in cart', () => {});

  it('Should remove items from cart', () => {});
});
