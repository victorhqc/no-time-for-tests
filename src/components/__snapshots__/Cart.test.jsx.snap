// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart render items from cart 1`] = `
<div>
  <ul>
    <li
      key="1"
    >
      foo
      : 
      US$ 10.00 (1)
      <button
        onClick={[Function]}
      >
        x
      </button>
    </li>
    <li
      key="2"
    >
      bar
      : 
      US$ 15.00 (2)
      <button
        onClick={[Function]}
      >
        x
      </button>
    </li>
  </ul>
  <h3>
    Total in Cart: 
    NaN
  </h3>
</div>
`;

exports[`Cart render total amount in cart 1`] = `
<div>
  <ul>
    <li
      key="1"
    >
      foo
      : 
      US$ 100.00 (2)
      <button
        onClick={[Function]}
      >
        x
      </button>
    </li>
  </ul>
  <h3>
    Total in Cart: 
    US$ 200.00
  </h3>
</div>
`;

exports[`Cart renders 1`] = `
<h3>
  Your cart is empty
</h3>
`;
